File: app/src/main/java/com/singh/gurtek/kotlinconductor/models/FlickerResponse.java
package com.singh.gurtek.kotlinconductor.models;

/**
 * Created by JacksonGenerator on 12/13/17.
 */

import com.fasterxml.jackson.annotation.JsonProperty;


public class FlickerResponse {
    @JsonProperty("stat")
    private String stat;
    @JsonProperty("photos")
    private Photos photos;
}

File: app/src/main/java/com/singh/gurtek/kotlinconductor/models/FlickerResponse.kt
package com.singh.gurtek.kotlinconductor.models

/**
 * Created by JacksonGenerator on 12/13/17.
 */

import com.fasterxml.jackson.annotation.JsonProperty


data class FlickerResponse(
        @JsonProperty("stat") val stat: String,
        @JsonProperty("photos") val photos: Photos?

)